src/main.cpp

```diff
@@ -6,15 +6,13 @@
  * https://discord.com/oauth2/authorize?client_id=940762342495518720&scope=bot+applications.commands&permissions=139586816064
  */
 
-using json = nlohmann::json;
-
 int main(int argc, char const *argv[])
 {
-    json configdocument;
+    dpp::json configdocument;
     std::ifstream configfile("../config.json");
     configfile >> configdocument;
 
-    /* Setup the bot */
+    /* Set up the bot */
```

Comment:
This is incorrect, "Setup" is right here.

Reply:
"Set up" is the verb, "setup" is a noun, the PR is right.

Reply:
But set up is used as a verb here, so it should be "set up", no?